packages/compiler-sfc/src/script/resolveType.ts

```diff
@@ -957,8 +957,10 @@ function resolveExt(filename: string, fs: FS) {
     tryResolve(filename) ||
     tryResolve(filename + `.ts`) ||
     tryResolve(filename + `.d.ts`) ||
+    tryResolve(filename + `.tsx`) ||
```

Comment:
Seems TypeScript would look up for `.tsx` before `.d.ts`:
https://www.typescriptlang.org/docs/handbook/modules/reference.html#file-extension-substitution

Reply:
Appreciate for your reminding. I'm not fimilar with behavior of TypeScript.
I have added a test case named "relative tsx" in `resolveType.spec.ts`. Very appreciate if you can check it for me.